kafka-connect-s3/src/test/java/io/confluent/connect/s3/storage/S3StorageTest.java

```diff
@@ -142,7 +142,7 @@ public void testUserDefinedCredentialsProvider() throws Exception {
     String configPrefix = S3SinkConnectorConfig.CREDENTIALS_PROVIDER_CONFIG_PREFIX;
     localProps.put(configPrefix.concat(DummyAssertiveCredentialsProvider.ACCESS_KEY_NAME), "foo_key");
     localProps.put(configPrefix.concat(DummyAssertiveCredentialsProvider.SECRET_KEY_NAME), "bar_secret");
-    localProps.put(configPrefix.concat(DummyAssertiveCredentialsProvider.CONFIGS_NUM_KEY_NAME), "3");
+    localProps.put(configPrefix.concat(DummyAssertiveCredentialsProvider.CONFIGS_NUM_KEY_NAME), "5");
```

Comment:
Unit tests were not running in jenkins job initially because of missing junit-vintage-engine dep.
After adding missing deps, some unit tests were failing.
Had to update this because we have added 2 more configs [here](https://github.com/confluentinc/kafka-connect-storage-cloud/blob/master/kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java#L817)

Reply:
@pkuntalteam1 please take a look if we need to change anything regarding this test.

Reply:
@arihant-confluent : looks good